### DISCUSSION TITLE
Problema con il mio ZCS Azzurro HYD 3/6k ZSS

### DISCUSSION THREAD

**Discussion by CapJS91** [davidrapan/ha-solarman#169] [Q&A] [answered]
![Screenshot 2024-09-22 172048](https://github.com/user-attachments/assets/505d0de9-5388-42fe-9250-868542303725)
Sto provando in tutti i modi a configurare il mio ZCS, ma nulla, non ne vuole sapere, non so piu' cosa fare.

**Comment by davidrapan** [comment]
Potresti abilitare la registrazione del debug e allegare qui il file di registro?

Modifica: prova a utilizzare `zcs_azzurro-hyd-zss-hp.yaml`

**Comment by CapJS91** [comment]
[home-assistant_solarman_2024-09-22T17-00-11.055Z.log](https://github.com/user-attachments/files/17090510/home-assistant_solarman_2024-09-22T17-00-11.055Z.log)

**Comment by CapJS91** [comment]
Il mio inverter è per zcs azzurro HYD6000 per accumulo

**Reply by davidrapan** [reply]
Sì, significa che il file che dovresti usare è `zcs_azzurro-hyd-zss-hp.yaml`

**Reply by davidrapan** [reply, marked as answer]
Hai ricevuto `OSError: [Errno 30] Read-only file system`

Devi accedere tramite ssh alla tua istanza HA ed eseguire `ha core rebuild`.

**That will fix your issue**.

**Reply by CapJS91** [reply]
Grazie mille finalmente ha funzionato :)

**Reply by davidrapan** [reply]
Prego! 😉
